### PR TITLE
Add all line breaking whitespace to splitLines and lineSplitter

### PR DESCRIPTION
**Pull request by Panke**
Adheres to http://www.unicode.org/versions/Unicode7.0.0/ch05.pdf

### CODE REVIEW

File: std/string.d [hunk 1]
```diff
@@ -2055,10 +2072,10 @@ S[] splitLines(S)(S s, in KeepTerminator keepTerm = KeepTerminator.no) @safe pur
     {
     foreach (S; TypeTuple!(char[], wchar[], dchar[], string, wstring, dstring))
     {
-        auto s = to!S("\rpeter\n\rpaul\r\njerry\u2028ice\u2029cream\n\nsunday\nmon\u2030day\n");
+        auto s = to!S("\rpeter\n\rpaul\r\njerry\u2028ice\u2029cream\n\nsunday\nmon\u2030day\nschadenfreude\vkindergarten\f\vcookies\u0085");
```
Comment (quickfur) [inline std/string.d]: This line is too long and hard to read. Could we please break it up into shorter lines?

Comment (Panke) [inline std/string.d]: ofc

File: std/string.d [hunk 2]
```diff
@@ -1988,6 +1989,8 @@ S[] splitLines(S)(S s, in KeepTerminator keepTerm = KeepTerminator.no) @safe pur
     {
         switch (s[i])
         {
+            case '\v':
+            case '\f':
             case '\n':
```
Comment (JakobOvrum) [inline std/string.d]: Same as:
```d
case '\v', '\f', '\n':
```

**Comment by JakobOvrum** [comment]
LGTM. This is a breaking change but it is also a bugfix, so that should be OK.

File: std/string.d [hunk 3]
```diff
@@ -1952,10 +1952,11 @@ S capitalize(S)(S s) @trusted pure
...
 /++
-    Split $(D s) into an array of lines using $(D '\r'), $(D '\n'),
-    $(D "\r\n"), $(XREF uni, lineSep), and $(XREF uni, paraSep) as delimiters.
-    If $(D keepTerm) is set to $(D KeepTerminator.yes), then the delimiter
-    is included in the strings returned.
+    Split $(D s) into an array of lines according to the unicode standard using
+    $(D '\r'), $(D '\n'), $(D "\r\n"), $(XREF uni, lineSep), 
+    $(XREF uni, paraSep), $(D U+0085) (NEL), $(D '\v')  and $(D '\f')
+    as delimiters. If $(D keepTerm) is set to $(D KeepTerminator.yes), then the
+    delimiter is included in the strings returned.
```
Comment (WalterBright) [inline std/string.d]: Add:
```
Adheres to http://www.unicode.org/versions/Unicode7.0.0/ch05.pdf
```

Comment (Panke) [inline std/string.d]: done

Comment (WalterBright) [inline std/string.d]: Thanks!

**Comment by WalterBright** [comment]
I agree it's a breaking change, but a very minor one and we need to follow the Unicode standard. I prefer to categorize it as a bugfix. I would have originally written it this way, but did not know that the Unicode standard had words on the topic.

**Pull request merged**